ui/pages/home/beta/beta-home-footer.component.js

```diff
@@ -43,7 +43,7 @@ const BetaHomeFooter = () => {
       </a>{' '}
       |{' '}
       <a href={SUPPORT_LINK} target="_blank" rel="noopener noreferrer">
-        {t('needHelpFeedback')}
+        {t('needHelpFeedback')} ajnddwjn
```

Comment:
typo here?

Reply:
Debugging something unrelated. Pushed this by mistake

Reply:
Removed in [0be748e](https://github.com/MetaMask/metamask-extension/pull/21222/commits/0be748e254e8daf73c6e8101e9aa0572f5d323ee)